src/core/thread/network_diagnostic.cpp

```diff
@@ -267,6 +267,14 @@ Error Server::AppendDiagTlv(uint8_t aTlvType, Message &aMessage)
         error = Tlv::Append<ModeTlv>(aMessage, Get<Mle::MleRouter>().GetDeviceMode().Get());
         break;
 
+    case Tlv::kEui64:
+    {
+        Mac::ExtAddress eui64;
+        Get<Radio>().GetIeeeEui64(eui64);
+        error = Tlv::Append<Eui64Tlv>(aMessage, eui64);
+        break;
+    }
+
```

Comment:
Should we also add support for parsing this TLV in a response and allow the value to be retarived from `otThreadGetNextDiagnosticTlv()` and `GetNextDiagTlv()`?

Reply:
Also would be good to update `cli` to support this TLV in `Interpreter::HandleDiagnosticGetResponse()`?